search/search.py

```diff
@@ -46,6 +46,20 @@ def trigram_search(query: str, queryset, fields: List[str]):
     return queryset.annotate(similarity=trigram_query).order_by('-similarity')
 
 
+def field_type(model, field):
+    """Determine a type name of Django field."""
```

Comment:
it'll be true OOP to create smth like this:
```
class FieldMeta:
	"""Django Model's field metadata."""
	
	TEXT_TYPES = ['CharField', 'TextField']
	
	def __init__(model, field: str):
		self.model = model
		self.field = field
		
	def type() -> str:
		# field_type() function code here
		...
		
	def is_text() -> bool:
		return self.type() in TEXT_TYPES
```

If you agree with it, pdd subtask would be good